#167749942 Integrate TravisCI with readme badge

alatos2 opened the pull request:
#### What does this PR do?

This PR integrates TravisCI with readme badge

#### Description of Tasks

- Setup travis.yml file to instruct travis on what to build
- Update README.md with travis-ci badge to display build passing or failing

#### Relevant pivotal tracker story

[167749942](https://www.pivotaltracker.com/story/show/167749942)

ascii-dev commented on .travis.yml:
```diff
@@ -0,0 +1,23 @@
+notifications:
+  email: false
+  
+after_success: npm run coverage
```
Kindly add a new line here. Thanks

alatos2 replied:
Correction well received

hustlaviola reviewed (approved):
Good job

alatos2 committed:
chore(travis-ci): Integrate travis-ci with readme badge

- run tests automatically.
- run other preliminary build steps required to ship the next iteration of the product.

[Maintains #167749942]

ascii-dev reviewed (approved):
LGTM
